src/auro-menu.js

```diff
@@ -39,31 +41,59 @@ class AuroMenu extends LitElement {
 
   firstUpdated() {
     this.addEventListener('keydown', this.handleKeyDown);
-    this.addEventListener('mousedown', this.handleClick);
-    this.addEventListener('click', this.selectedOption);
+    this.addEventListener('click', this.makeSelection);
   }
 
   /**
-   * Manage click events.
-   * Event fires, but attributes are not changed on elements with the disabled attribute.
+   * Reset all attributes on all menuoptions.
    * @private
-   * @param {Object} event - Event object from the browser.
    */
-  handleClick(event) {
+  resetOptionsStates() {
+    this.items.forEach((item) => {
+      item.tabIndex = -1;
+      item.removeAttribute('selected');
+      item.removeAttribute('aria-selected');
+    });
+  }
 
-    this.index = this.items.indexOf(event.target);
+  /**
+   * Set the attributes on the selected menuoption, the menu value and stored option.
+   * @param {Object} option - The menuoption to be selected.
+   * @private
+   */
+  handleLocalSelectState(option) {
+    option.tabIndex = 0;
+    option.setAttribute('selected', '');
+    option.ariaSelected = true;
```

Comment:
Is the intention with the first load that the first element be selected if there isn't a defined pre-selected option? That's not a spec that I am aware of?

Reply:
The previous version was only selecting the first option on an initial click event in a special case with an attribute. The current `main` branch does not do this on render. 

https://auro.alaskaair.com/components/auro/menu